config/initializers/devise.rb

```diff
@@ -0,0 +1,311 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '2abd872e5842d3077612dab522ff010c4ac84447b5ffc026490256b95b93813cf08589ef9397865e1afe03d621a7e6d97953f0c94d90d3fdf708a6d04234f2c5'
+
+  # ==> Controller configuration
+  # Configure the parent class to the devise controllers.
+  # config.parent_controller = 'DeviseController'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+
```

Comment:
- [ ] When I sign up for a new account I see the confirmation message which supposes to be sent to the registered email on the screen directly and this is not supposed to happen, So please when you click sign up make the confirmation message sent to the email you registered with.

[Hint]: See this [video ](https://youtu.be/sVJRkTxqlSQ?t=344)to know how to do it. (I send a link with the specific time talking about the mail sender but I suggest to see the whole video for better understanding)